<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModelWidget</class>
 <widget class="QWidget" name="ModelWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>312</width>
    <height>215</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="geometryTab">
      <attribute name="title">
       <string>Geometry</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QTreeView" name="geometryTreeView"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="materialsTab">
      <attribute name="title">
       <string>Materials</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QComboBox" name="materialsComboBox"/>
       </item>
       <item row="1" column="0">
        <widget class="MaterialWidget" name="materialWidget" native="true"/>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="animationsTab">
      <attribute name="title">
       <string>Animations</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QComboBox" name="animationsComboBox"/>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>120</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MaterialWidget</class>
   <extends>QWidget</extends>
   <header>nodeitemdetailwidget/modelwidget/materialwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
